.eslintrc

```diff
@@ -42,8 +42,6 @@
     "no-multiple-empty-lines": [2, {"max": 1}],
     // Disallow the use of ternary operators when a simpler alternative exists.
     "no-unneeded-ternary": 2,
-    // Avoid code that looks like two expressions but is actually one.
-    "no-unexpected-multiline": 2,
```

Comment:
This beast seems to contradict the current clang-format that was updated earlier today. We should either leave this out or change the clang-format rules...